packages/oc-template-react-compiler/lib/to-abstract-base-template-utils/oc-webpack/lib/configurator/client/index.js

```diff
@@ -19,30 +19,42 @@ module.exports = options => {
     ? "oc__[path][name]-[ext]__[local]__[hash:base64:8]"
     : "[local]__[hash:base64:8]";
 
+  const cssModuleLoader = {
+    loader: require.resolve("css-loader"),
+    options: {
+      importLoaders: 1,
+      modules: true,
+      localIdentName,
+      camelCase: true
+    }
+  };
+
+  const postCssModuleLoader = {
+    loader: require.resolve("postcss-loader"),
+    options: {
+      ident: "postcss",
+      plugins: [
+        require("postcss-import"),
+        require("postcss-extend"),
+        require("postcss-icss-values"),
+        require("autoprefixer")
+      ]
+    }
+  };
+
   const cssLoader = {
     test: /\.css$/,
+    use: [MiniCssExtractPlugin.loader, cssModuleLoader, postCssModuleLoader]
+  };
+
+  const sassLoader = {
+    test: /\.scss$/,
     use: [
       MiniCssExtractPlugin.loader,
+      cssModuleLoader,
+      postCssModuleLoader,
       {
-        loader: require.resolve("css-loader"),
-        options: {
-          importLoaders: 1,
-          modules: true,
-          localIdentName,
-          camelCase: true
-        }
-      },
-      {
-        loader: require.resolve("postcss-loader"),
-        options: {
-          ident: "postcss",
-          plugins: [
-            require("postcss-import"),
-            require("postcss-extend"),
-            require("postcss-icss-values"),
-            require("autoprefixer")
-          ]
-        }
+        loader: "sass-loader"
```

Comment:
Shouldn't we have `require.resolve("sass-loader")` here to be sure all the paths are supported?